examples/gnrc_minimal/Makefile

```diff
@@ -13,7 +13,7 @@ BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h
 #USEMODULE += xbee
 
 ## set default UART to use in case none was defined
-#XBEE_UART ?= "UART_NUMOF-1"
+#XBEE_UART ?= "1"
```

Comment:
Well I find that the most common usage of XBee is on platforms which offer more than one UART, especially for Arduino boards, on which the XBee shield is attached to UART0. 

Using the current approach the XBee goes to UART3 in arduino-mega2560, which is not evident to know IMHO. Additionally, on platforms with only 1 UART it goes to UART0, which is usually used for STDIO, thus conflicting since we don't have any check in the current master to avoid such conflict. 

Thus, I prefer to state exactly which UART I'm going to use, and if it doesn't exists at least I'll have STDIO free.